### PR TITLE
Optionally convert numbers in CSV files

### PR DESCRIPTION
**Pull request by fritzgrabo**
This is the `dsq` part of https://github.com/multiprocessio/datastation/issues/265.

Make sure to merge the DataStation side first 😅

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -481,6 +481,47 @@ dsq> SELECT * FROM {} WHERE NAME = 'Kevin';
...
+
+This can lead to unexpected results in queries. Consider the following
+example:
+
+```
+$ cat scores.csv
+name,score
+Fritz,90
+Rainer,95
+Fountainer,100
```
Comment (eatonphil) [inline README.md:497]: To demonstrate both integers and floats could you make one of these numbers a float? I.e. with a .0 suffix. Thanks!

**Comment by eatonphil** [comment]
Just one small nit for the docs but otherwise looks good! Update the datastation commit and will merge and release!

File: README.md (hunk 1 above)
Comment (fritzgrabo) [reply README.md:497]: Updated 👍

**Comment by fritzgrabo** [comment]
Updated the `datastation/runner` dependency to point to the latest commit on `main` (`405d8c1`) -- this builds cleanly now 👍

**Comment by eatonphil** [comment]
Awesome, thanks!

**Pull request merged**